app/src/main/res/layout/fragment_instance_list.xml

```diff
@@ -46,6 +46,6 @@
         android:focusable="true"
         app:backgroundTint="?attr/colorPrimary"
         app:fabSize="auto"
-        app:srcCompat="@drawable/ic_add" />
```

Comment:
I think this file can be reformatted with `Ctrl+Shift+Alt+L`, since the order of attributes here is not alphabetical (maybe other files, too)